src/content/apis/cryptocurrency/brave-new-coin.md

```diff
@@ -0,0 +1,10 @@
+---
+name: Brave New Coin
+description: Providing data, indexes, analysis and insight on the cryptographic asset marketplace
+categories:
+  - cryptocurrency
+features:
+  https: true
+url: https://docs.bravenewcoin.com/
+logo: https://docs.bravenewcoin.com/images/logo.png

```

Comment:
The logo has to be downloaded and place inside the repository itself.
Please check the contributing guides for more details.